### ISSUE TITLE
Feature Request: Add flag to specify location of slashing protection database file

### ISSUE THREAD

**Issue by djoyahoy** [sigp/lighthouse#2273]
## Description

I would like a flag on the validator client that allows me to specify where the slashing database file is written. This would allow me to separate the slashing database from sensitive keystore data. 

## Version

N/A

## Present Behaviour

Currently, the database is written to the validators dir, which also contains the definitions file and keystore data.

## Expected Behaviour

The flag would allow the database to written to a location of my choosing.

## Steps to resolve

N/A

**Comment by michaelsproul** [comment]
Good idea! I think this is something we could safely implement. We'll look into it next week

**Comment by michaelsproul** [comment]
@djoyahoy Did you mean to close this? If this is still a feature you need, best to leave the issue open until we actually implement it 😅

**Comment by djoyahoy** [comment]
@michaelsproul my mistake. Thank you!

**Comment by paulhauner** [comment]
This might be a good spare-time task for @realbigsean :slightly_smiling_face:
